### PR TITLE
Make Courses#26194

### PR DESCRIPTION
**Pull request by DucNguyenVan**
add users + subjects + time

### CODE REVIEW

File: app/views/admin/courses/new.html.erb [hunk 1]
```diff
@@ -20,7 +24,17 @@
           <%= builder.hidden_field :id, value: nil %>
         <% end %>
       <% end %>
+
+      <h3>Start date</h3>
+      <%= f.date_select :start_time, :start_year => Date.current.year,
+                   :end_year => Date.current.year + 5 , class: 'form-control' %>
```
Comment (dieunb) [inline app/views/admin/courses/new.html.erb]:
```
end_year: Date.current.year + 5
```

File: app/views/admin/courses/new.html.erb [hunk 2]
```diff
@@ -20,7 +24,17 @@
...
+      <h3>Finish date</h3>
+      <%= f.date_select :finish_time, :start_year => Date.current.year,
```
Comment (dieunb) [inline app/views/admin/courses/new.html.erb]:
```
start_year: Date.current.year
end_year: Date.current.year + 5
```

File: app/views/admin/courses/new.html.erb [hunk 3]
```diff
@@ -20,7 +24,17 @@
           <%= builder.hidden_field :id, value: nil %>
         <% end %>
       <% end %>
+
+      <h3>Start date</h3>
+      <%= f.date_select :start_time, :start_year => Date.current.year,
```
Comment (dinhvantiep) [inline app/views/admin/courses/new.html.erb]: Đức chỗ này sai convention thì phải. 
-> start_year: Date.current.year

File: app/views/admin/courses/edit.html.erb [hunk 4]
```diff
@@ -24,6 +26,13 @@
           <%= builder.hidden_field :id, value: @course.subject_course_id(subject) %>
         <% end %>
       <% end %>
+
+      <h3>Start date</h3>
+      <%= f.date_select :start_time, :start_year => Date.current.year,
```
Comment (dieunb) [inline app/views/admin/courses/edit.html.erb]:
```
start_year: Date.current.year
end_year: Date.current.year + 5
```

File: app/views/admin/courses/edit.html.erb [hunk 5]
```diff
@@ -24,6 +26,13 @@
...
+                   :end_year => Date.current.year + 5 , class: 'form-control' %>
+      <h3>Finish date</h3>
+      <%= f.date_select :finish_time, :start_year => Date.current.year,
```
Comment (dieunb) [inline app/views/admin/courses/edit.html.erb]:
```
```
start_year: Date.current.year
end_year: Date.current.year + 5
```
```

File: app/assets/stylesheets/application.css [hunk 6]
```diff
@@ -13,3 +13,4 @@
  *= require_tree .
  *= require_self
  */
+  *= require bootstrap-datepicker
```
Comment (dieunb) [inline app/assets/stylesheets/application.css]: require wrong

**Pull request merged**